src/main/resources/scripts/teleport with compass.sk

```diff
@@ -3,14 +3,8 @@
 # with a compass like in WorldEdit.
 #
 
-aliases:
-	# blocks without collision as of Minecraft 1.5 (excluding fire, nether portal & end portal)
-	nonsolid = 0, 6, 27, 28, 30, 31, 32, 37, 38, 39, 40, 50, 55, 59, 63, 64, 65, 66, 68, 69, 70, 71, 72, 75, 76, 77, 78:0, 83, 85:4-7, 104, 105, 106, 111, 115, 131, 132, 141, 142, 143, 147, 148, 157
-
 on rightclick with compass:
 	player has permission "skript.teleport"
 	loop blocks above targeted block:

```

Comment:
But how ``nonsolid`` blocks are usefull here? I mean, I did tests without this condition and it works really well.

Reply:
Does it not prevent the player from being teleported inside a block ?
Also I think you should format the condition with `if` like you did in other examples

Reply:
When you use ``if`` word, you are supposed to follow with the ``else`` part, or you just use it as "code block" so here you are not supposed to use the ``if`` word because it's an "affirmative" condition (so if this condition is not respected, it will not continue the code)

Reply:
Yeah that's what I do but I would write it there as a convention, in Java or Python you cannot omit it